backend/entities/courses/section_entity.py

```diff
@@ -4,6 +4,8 @@
 from sqlalchemy import Integer, String, DateTime, ForeignKey
 from sqlalchemy.orm import Mapped, mapped_column, relationship
 
+from backend.models.room_assignment_type import RoomAssignmentType
```

Comment:
Should we prefer relative import paths? (Nit)

Reply:
Agree @KrisJordan! We can do this - automatically importing using VSCode uses absolute paths, but I try to convert them where I can.